### PR TITLE
du: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/du.md [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
-- Lista los tamaños de un directorio y sus subdirectorios en las unidades dadas (B/KiB/MiB):
+- Lista los tamaños de un directorio y sus subdirectorios, en la unidad dada (B/KiB/MiB):
 
 `du -{{b|k|m}} {{ruta/al/directorio}}`
 
-- Lista los tamaños de un directorio y sus subdirectorios en formato legible para humanos (es decir, seleccionando automáticamente las unidades apropiadas para cada tamaño):
+- Lista los tamaños de un directorio y sus subdirectorios, de forma legible (es decir, seleccionando automáticamente la unidad adecuada para cada tamaño):
 
-`du -h {{ruta/al/directorio}}`
+`du -h {{ruta/a/directorio}}`
```
Comment (ikks) [inline pages.es/common/du.md:12]: let's unify {{ruta/al/directorio}}

File: pages.es/common/du.md [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 
-- Muestra el tamaño de un solo directorio en formato legible para humanos:
+- Muestra el tamaño de un único directorio, en unidades legibles para el ser humano:
 
-`du -sh {{ruta/al/directorio}}`
+`du -sh {{ruta/a/directorio}}`
```
Comment (ikks) [inline pages.es/common/du.md:16]: let's unify {{ruta/al/directorio}}

File: pages.es/common/du.md [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
 
-- Lista los tamaños legibles para humanos de un directorio y de todos los archivos y directorios dentro del mismo:
+- Lista los tamaños legibles de un directorio y de todos los archivos y directorios que contiene:
 
-`du -ah {{ruta/al/directorio}}`
+`du -ah {{ruta/a/directorio}}`
```
Comment (ikks) [inline pages.es/common/du.md:20]: let's unify {{ruta/al/directorio}}

File: pages.es/common/du.md (hunk 2 above)
Comment (ikks) [reply pages.es/common/du.md:16]: and remove "para el ser humano" to make it more terse, the other ones are more readable too :) .

File: pages.es/common/du.md [hunk 4]
```diff
@@ -1,28 +1,32 @@
...
 
-- Lista los tamaños legibles para humanos de un directorio y sus subdirectorios hasta N niveles de profundidad:
+- Lista los tamaños legibles de un directorio y sus subdirectorios, hasta N niveles de profundidad:
 
-`du -h --max-depth=N {{ruta/al/directorio}}`
+`du -h --max-depth=N {{ruta/a/directorio}}`
```
Comment (ikks) [inline pages.es/common/du.md:24]: let's unify {{ruta/al/directorio}}

**Review by ikks — APPROVED** [review]

File: pages.es/common/du.md [hunk 5]
```diff
@@ -1,28 +1,32 @@
...
 
 `du -{{b|k|m}} {{ruta/al/directorio}}`
 
-- Lista los tamaños de un directorio y sus subdirectorios en formato legible para humanos (es decir, seleccionando automáticamente las unidades apropiadas para cada tamaño):
+- Lista los tamaños de un directorio y sus subdirectorios, de forma legible (es decir, seleccionando automáticamente la unidad adecuada para cada tamaño):
 
 `du -h {{ruta/al/directorio}}`
 
-- Muestra el tamaño de un solo directorio en formato legible para humanos:
+- Muestra el tamaño de un único directorio, en unidades legibles para el ser humano:
```
Comment (ikks) [inline pages.es/common/du.md:14]: I would vouch for removing "para el ser humano", to make the read more terse. Let me know if you agree.

**Review by karthik-script — APPROVED** [review]

**Pull request merged**